src/assets/thirdPartyServices/snet/cntk-image-recon/image_recon_pb.js

```diff
@@ -0,0 +1,355 @@
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+
+var jspb = require('google-protobuf');
+var goog = jspb;
+var global = Function('return this')();
+
+goog.exportSymbol('proto.Input', null, global);
+goog.exportSymbol('proto.Output', null, global);
```

Comment:
I forgot to mention, we need to generate the stubs with name spaces. We can follow the convention as follows 
`${package_name}_${org_id}_${service_id}`. Need to change this in example service too.

Reply:
[done](https://github.com/singnet/refactored-dapp/pull/120/commits/719d5cd99575b7869e6b89f4ada093ab45cf5216)